[FIX] Default value when no value found

diggy128 opened the pull request:
Default value should be set to Empty record instead of False for
Many2one fields when no value is to be set.
See: https://github.com/odoo/odoo/issues/4384

anajuaristi commented:
Hi @diggy128. Thank you for your contribution. 

@oihane @Daniel-CA
Could you please review this PR? I'm not sure it's correct since add middle staff should not show any value but let user to select the product he wants from full catalogue.

Please let me know

oihane commented:
@diggy128 can you please rebase this PR?

Thanks for your contribution, let's see if the travis fixes.

Daniel-CA commented:
:+1:  LGTM, waiting for rebase.

Commit:
[FIX] Default value when no value found

Default value should be set to Empty record instead of False for
Many2one fields when no value is to be set.
See: https://github.com/odoo/odoo/issues/4384

diggy128 commented:
Sorry for the mess. my zenbook's touchpad is freaking out and pissing me off. 
Should be OK now.

Commit:
Merge branch '8.0' into 8.0
